packages/dds/tree/src/core/tree/anchorSet.ts

```diff
@@ -131,6 +131,30 @@ export interface AnchorEvents {
 	 */
 	subtreeChanging(anchor: AnchorNode): PathVisitor | void;
 
+	/**
+	 * Emitted after the subtree rooted at `anchor` may have been changed.
+	 * 
+	 * @remarks
+	 * While this event is always emitted in the presence of changes to the subtree,
+	 * it may also be emitted even though no changes have been made to the subtree.
+	 * May be emitted multiple times within the application of a single edit or transaction.
+	 *
+	 * @privateremarks
+	 * The implementation details of the delta visit traversal have a big effect on how/when this event is emitted.
+	 * As of 2024-03-27, it is emitted on a node as the delta traversal exits that node, which by design of the current
+	 * traversal algorithm happens after all changes to the node's children have been applied, so this event is only
```

Comment:
```suggestion
	 * traversal algorithm happens after some changes to the node's children have been applied, so this event is only
```

Reply:
I guess technically we're both incorrect, since traversal can still go through a node and do nothing in it during one of the two passes 😝 but taking the suggestion and I'll probably try to clarify more when I move this.